src/Pages/Login/Login.js

```diff
@@ -38,21 +39,24 @@ class Login extends React.Component {
     this.goToSignUpPage();
   };
 
-  // async requestLogin() {
-  //   try {
-  //     const res = await fetch('#', {
-  //       method: '',
-  //       body: JSON.stringify({
-  //         email: '',
-  //         password: '',
-  //       }),
-  //     });
+  async requestLogin() {
+    const { email, password } = this.state;
+    try {
+      const res = await fetch('http://10.58.7.23:8000/users/login', {
```

Comment:
ip 주소나 포트 등이 수정될 경우 해당 주소로 요청하는 로직 모든 곳을 돌아다니며 수정해야할 필요가 생길 것 같습니다. src/config.js 파일을 만드시고 그 안에서 아래와 같이 사용할 수 있도록 변수를 관리하는게 어떨까요?
```suggestion
      const res = await fetch(`{API}/users/login`, {
```

Reply:
네 확인했습니다. 수정하겠습니다.